packages/api/cms-api/src/index.ts

```diff
@@ -59,6 +59,7 @@ export { RootBlockEntity } from "./blocks/decorators/root-block-entity";
 export { EmailLinkBlock } from "./blocks/email-link.block";
 export { ExternalLinkBlock } from "./blocks/ExternalLinkBlock";
 export { ColumnsBlockFactory } from "./blocks/factories/columns-block.factory";
+export { BlocksBlockInputInterface } from "./blocks/factories/createBlocksBlock";
```

Comment:
Why is this export needed?

Reply:
@johnnyomair  There are other factory methods like `createOptionalBlock` or `createSeoBlock` (as an example) which export their interfaces, i thought i would make sense.

We can as well make the BlocksBlockInputInterface private.